### PR TITLE
Updated znc to 1.7.5

### PR DESCRIPTION
**Pull request by andreas-becker**
_Motivation:_  Update znc because there has been a remote code execution bug in previous versions
_Linked issues:_  http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12816

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

I did try to compile the packages with the build rule `all-supported` but wasn't able to do so in the docker container. I successfully generated a package for armada370 and was able to install it on my device.

### CODE REVIEW

File: spk/znc/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = znc
-SPK_VERS = 1.7.1
-SPK_REV = 12
+SPK_VERS = 1.7.5
+SPK_REV = 8
```
Comment (hgy59) [inline spk/znc/Makefile]: please increment the spk revision to the former one (use 13)

File: spk/znc/Makefile [hunk 2]
```diff
@@ -12,7 +12,7 @@ DESCRIPTION_FRE = Bouncer IRC avancé. Un bouncer IRC n\’est en fait qu\’un
 DESCRIPTION_SPN = IRC bouncer avanzado. Un IRC bouncer no es más que un proxy para IRC. ZNC estará siempre conectado a tus canales, y hará de puerta de enlace entre tus clientes y tus servidores IRC. Puedes, por ejemplo, consultar mensajes mientras estuviste desconectado o esconder tu identidad.
 RELOAD_UI = yes
 DISPLAY_NAME = ZNC
-CHANGELOG = "1. Update to ZNC 1.7.1<br/>2. Update modules<br/>3. Add generic DSM5+6 service support"
+CHANGELOG = "Update to ZNC 1.7.5"
 BETA = 1
```
Comment (hgy59) [inline spk/znc/Makefile]: maybe it's time to remove BETA?

Comment (ymartin59) [reply spk/znc/Makefile]: I agree

**Comment by andreas-becker** [comment]
Should i change the MAINTAINER field as well?

**Pull request merged**